.github/workflows/integration_test.yaml

```diff
@@ -1,10 +1,28 @@
+
 name: Integration tests
 
 on:
   pull_request:
 
 jobs:
+  extra-args:
+    runs-on: ubuntu-latest
+    outputs:
+      args: ${{ steps.flags.outputs.args }}
+    steps:
+      - name: Determine extra args
+        id: flags
+        env:
+          TITLE: ${{ github.event.pull_request.title }}
+        run: |
+          EXTRA_ARGS="--crash-dump=on-failure"
+          if [[ "$TITLE" == *"[COS]"* ]]; then
+            EXTRA_ARGS="$EXTRA_ARGS --cos"
+          fi
+          echo "args=$EXTRA_ARGS" >> "$GITHUB_OUTPUT"
```

Comment:
So, this is clever.  if COS is in the title we run the cos tests.  nice

Reply:
That's correct, thanks!!